pkg/identity/identitysync/okta.go

```diff
@@ -19,12 +20,14 @@ type OktaSync struct {
 	client   *okta.Client
 	orgURL   gconfig.StringValue
 	apiToken gconfig.SecretStringValue
+	groups   gconfig.OptionalStringValue
 }
 
 func (s *OktaSync) Config() gconfig.Config {
 	return gconfig.Config{
 		gconfig.StringField("orgUrl", &s.orgURL, "the Okta organization URL"),
 		gconfig.SecretStringField("apiToken", &s.apiToken, "the Okta API token", gconfig.WithNoArgs("/granted/secrets/identity/okta/token")),
+		gconfig.OptionalStringField("groups", &s.groups, "Groups that users are synced from."),
```

Comment:
I would rename this option, instead of just `groups`, something more explicit like `user_groups`.

In this implementation we are still reading all the groups in `ListGroups`, but getting the users 
 
```suggestion
		gconfig.OptionalStringField("user_groups", &s.userGroups, "Groups that users are synced from."),
```